server-spi/src/main/java/org/keycloak/models/UserModel.java

```diff
@@ -300,7 +300,7 @@ default long getGroupsCountByNameContaining(String search) {
 
     enum RequiredAction {
         VERIFY_EMAIL, UPDATE_PROFILE, CONFIGURE_TOTP, UPDATE_PASSWORD, TERMS_AND_CONDITIONS,
-        VERIFY_PROFILE
+        VERIFY_PROFILE, GRANT_MANAGEMENT
```

Comment:
Considering that Grant Management might be preview feature at first, it might be better not to include it inside `UserModel` like `WebAuthnRegister` required action.